lib/aion_cli/commands/data.rb

```diff
@@ -208,7 +209,7 @@ def stats_3f(path)
             area_voters = rows.select { |row| row[index_area_number] == area_number && row[index_department] == department}
             area_label = "#{area_number} #{area_text}"
 
-            eligible_voters = area_voters.size
+            eligible_voters = total_json["Områdenr Afdelingsnr"]["#{area_number} #{department}"] || 0
```

Comment:
```suggestion
            eligible_voters = total_json.dig("Områdenr Afdelingsnr", "#{area_number} #{department}") || 0
```

Should `Områdenr Afdelingsnr` be a mapped value?